### ISSUE TITLE
Record and release video based on `VIDEO_2_SCRIPT.md`

### ISSUE THREAD

**Issue by derberg** [asyncapi/training#41]
We need to finally record https://github.com/asyncapi/training/blob/master/VIDEO_2_SCRIPT.md

Requirements:
- great quality video
- based on https://github.com/asyncapi/training/blob/master/VIDEO_2_SCRIPT.md and great animations already prepared in past by Barbanio and Tahi and storyboard https://docs.google.com/presentation/d/1mdkh2LxOAzcB_V8Kbtq0L_qyCG1tbG-u/edit#slide=id.p1
- speaker visible with a good background
- good and clear English pronunciation
- excellent sound quality
- get the video published under AsyncAPI YT account, under a dedicated playlists, like maybe "AsyncAPI Onboarding"
- get in touch with @iambami on getting proper promotion around releasing the video (we do not need to release video at the same time we promote it)

Nice to have:
- once a single video is released, would be great to explore creation of YT Shorts, like short by script section. Coordinate with @iambami promotion of these

**Comment by aeworxet** [comment]
How about using an AI-generated voice to maintain an accent-neutral pronunciation

[luvvoice.com-english-us-female-michelle](https://www.sendgb.org/view_file.php?id=2294&filename=luvvoice.com-english-us-female-michelle.mp3)
[luvvoice.com-english-us-male-brian](https://www.sendgb.org/view_file.php?id=2295&filename=luvvoice.com-english-us-male-brian.mp3)

https://luvvoice.com/auth/language/english

along with a drawn whiteboard explainer video

https://99designs.com/blog/video-animation/styles-of-video

instead of shooting a real one?

**Comment by aeworxet** [comment]
### Bounty Issue's service comment

Text labels: `bounty/2025-Q1`, `bounty/advanced`, `bounty/docs`
First assignment to regular contributors: [2024-12-20 00:00:00 UTC+12:00](https://time.lol/#2024-12-20T00:00:00+12:00)
End Of Life after: [2025-01-31 23:59:59 UTC-12:00](https://time.lol/#2025-01-31T23:59:59-12:00)

@asyncapi/bounty_team

##### The Bounty Program is not a Mentorship Program. The accepted level of Bounty Program Participants is **Middle**/**Senior**.
##### Regular contributors should explain in meaningful words how they are going to approach the resolution process when expressing a desire to work on this Bounty Issue.

**Comment by TenzDelek** [comment]
hi @derberg , 
I am also submitting my proposal here because both the #40 and this is kind of same task, so I would love to take on both task if permitted. below is the same content I wrote for #40 . 

I'm thrilled to express my genuine excitement and enthusiasm for the unique task under the bounty program you've mentioned. As a passionate videographer and camera enthusiast, I believe I can bring a fresh and engaging perspective to this project.

My experience with the equipment like Sony A6400 and proficiency in video editing software like Premiere Pro and After Effects will be invaluable in creating compelling and professional-quality content. Moreover, my skills in 3D animation using Blender can add an extra layer of visual appeal, helping to illustrate complex workflows in a clear and engaging manner.
my yt: https://www.youtube.com/@10zDelek/videos
my behance (where i post 3d): https://www.behance.net/tenzindelek?tracking_source=search_projects%7Ccybercity

> side note: I have a speech impediment (Stammering), but I assure you that it wont be a hinderance because I can control it if I am speaking alone.

I'm genuinely passionate about contributing to the growth of our community through this unique opportunity. The chance to combine my love for cameras, videography, and 3D animation (design) with the goal of creating valuable content for others is truly exciting.
If you need any further information about my approach or have any questions, please don't hesitate to ask. I'm eager to discuss this project in more detail and demonstrate how I can bring your vision to life.
Thank you for considering my application. I'm looking forward to the possibility of working on this task and making a meaningful contribution to AsyncAPI community.

**Comment by AceTheCreator** [comment]
Hey folks! I'm excited to inform y'all that i'm willing to work on the recording of this video.

**Comment by aeworxet** [comment]
### Bounty Issue's Timeline
|Complexity Level|Assignment Date (by GitHub)|Start Date (by BP Rules)|End Date (by BP Rules)|Draft PR Submission|Final PR Merge Start|Final PR Merge End|
|----------------|---------------------------|------------------------|----------------------|-------------------|--------------------|------------------|
|Medium          |2024-12-23              |[2025-01-06](https://time.lol/#2025-01-06T00:00:00+12:00)              |**[2025-03-02](https://time.lol/#2025-03-02T23:59:59-12:00)**        |[2025-01-26](https://time.lol/#2025-01-26T23:59:59-12:00)         |[2025-02-16](https://time.lol/#2025-02-16T23:59:59-12:00)         |**[2025-03-02](https://time.lol/#2025-03-02T23:59:59-12:00)**        |

##### Please note that the dates given represent deadlines, not specific dates, so if the goal is reached sooner, it's better.
##### Keep in mind the [responsibility](https://github.com/asyncapi/community/pull/897/files#diff-25ecb20a61754c225d6511ca08d7e7c9a14b9ca5a93e89bd42331e51c9ebf26dR226) for violations of the [Timeline](https://github.com/asyncapi/community/pull/897/files#diff-25ecb20a61754c225d6511ca08d7e7c9a14b9ca5a93e89bd42331e51c9ebf26dR99).
#
Assignee: @AceTheCreator (`githubID`: [40604284](https://api.github.com/user/40604284))

**Comment by aeworxet** [comment]
Short example video was [recorded](https://drive.google.com/file/d/101CzQQrpd51cHu6Py84DkjoNVtIgKWfN/view?usp=drive_link) for https://github.com/asyncapi/training/issues/40 to gather feedback on whether this is the correct direction to move in.

**Comment by aeworxet** [comment]
AsyncAPI Maintainer (@derberg, `githubID`: [6995927](https://api.github.com/user/6995927)) delayed a response critical for the technical resolution of the Bounty Issue on GitHub for two periods of three consecutive working days
https://github.com/asyncapi/training/issues/40#issuecomment-2615860699
https://github.com/asyncapi/training/issues/40#issuecomment-2636542820
so all remaining target dates of the Bounty Issue's Timeline are extended by two calendar weeks.

### Bounty Issue's Timeline Extended
|Complexity Level|Assignment Date (by GitHub)|Start Date (by BP Rules)|End Date (by BP Rules)|Draft PR Submission|Final PR Merge Start|Final PR Merge End|
|----------------|---------------------------|------------------------|----------------------|-------------------|--------------------|------------------|
|Advanced        |2024-12-23                 |[2025-01-06](https://time.lol/#2025-01-06T00:00:00+12:00)              |**[2025-03-16](https://time.lol/#2025-03-16T23:59:59-12:00)**        |[2025-02-09](https://time.lol/#2025-02-09T23:59:59-12:00)         |[2025-03-02](https://time.lol/#2025-03-02T23:59:59-12:00)         |**[2025-03-16](https://time.lol/#2025-03-16T23:59:59-12:00)**        |

##### Please note that the dates given represent deadlines, not specific dates, so if the goal is reached sooner, it's better.
##### Keep in mind the [responsibility](https://github.com/asyncapi/community/pull/897/files#diff-25ecb20a61754c225d6511ca08d7e7c9a14b9ca5a93e89bd42331e51c9ebf26dR224) for violations of the [Timeline](https://github.com/asyncapi/community/pull/897/files#diff-25ecb20a61754c225d6511ca08d7e7c9a14b9ca5a93e89bd42331e51c9ebf26dR99).
#
Assignee: @AceTheCreator (`githubID`: [40604284](https://api.github.com/user/40604284))

**Comment by aeworxet** [comment]
@AceTheCreator (`githubID`: [40604284](https://api.github.com/user/40604284)), please provide an update on the Bounty Issue.

**Comment by aeworxet** [comment]
A new example video for gathering feedback was [added](https://github.com/asyncapi/training/issues/40#issuecomment-2672241015) in the neighbouring issue.

**Comment by aeworxet** [comment]
Due to the specifics of this Bounty Issue, which is reflected in acting, recording, video editing, drawing animation, rendering animation, attaching subtitles, and video re-editing for the sake of synchronization all together again, the timeframe of execution of this Bounty Issue is less predictable than that of others.

**Comment by aeworxet** [comment]
The resolution of this Bounty Issue is still affected by hardships described in https://github.com/asyncapi/training/issues/41#issuecomment-2697490614.

**Comment by AceTheCreator** [comment]
Hey Folks! Thanks for the patience @aeworxet. 

Moving forward i can spend more time creating this videos as needed. Below is the link to the first one and I'll provide the second one this following week.

@derberg your feedback will be appreciated 🙏🏽


https://drive.google.com/file/d/1LqkyP0iiBcIyCJW8YEuwI4RPI71dYg9R/view?usp=sharing

**Comment by AceTheCreator** [comment]
Heyo! @derberg 

Here's the the link for the second video https://drive.google.com/file/d/1yrrJoT6Dtm4iUZenxuFtzP-EWCn99oEH/view?usp=sharing

Let me know what you think.

CC @aeworxet

**Comment by aeworxet** [comment]
@AceTheCreator 

> Only 1 remark: I don't see animation for EDA being used

How much time would you **realistically** need to address this issue?

**Comment by aeworxet** [comment]
@AceTheCreator 

> Only 1 remark: I don't see animation for EDA being used

How much time would you **realistically** need to address this matter?

**Comment by AceTheCreator** [comment]
> great stuff, well integrated animations. Only 1 remark: I don't see animation for EDA being used

Yeah, that was intentional. I found it challenging to use the animation directly within the paragraph for a few reasons:

- The animation runs longer than the paragraph itself.
- It didn’t quite sync well with the explanation in that context, which made it feel a bit off.
- Tried trimming the animation to fit the paragraph but i couldn't help but cut out key parts, making the edits obvious and less fluid, unlike the other animations I used.

**Comment by derberg** [comment]
how about just the part with ordering from home and showing drone - it just feels weird without, and leaves a questions - how different it is comparing to serverless, you know what I mean?

**Comment by AceTheCreator** [comment]
So you mean i should replace the illustration with `the part with ordering from home and showing drone`?

**Comment by AceTheCreator** [comment]
If yes, were in the section do you think i can use the EDA illustration? or i should just completely omit it?

**Comment by derberg** [comment]
the illustration that starts at 8:55 is great
how about pizza animation before also replacing animation visible around 8:10, your're gaining like extra 50sec, you could start pizza animation at 8:05

thoughts?

**Comment by AceTheCreator** [comment]
> the illustration that starts at 8:55 is great how about pizza animation before also replacing animation visible around 8:10, your're gaining like extra 50sec, you could start pizza animation at 8:05
> 
> thoughts?

Okay. let me try and fit in and see how it turns out 👍🏾

**Comment by derberg** [comment]
and please remember that each video of yours not only have your name and role, but also contact points

**Comment by aeworxet** [comment]
This Bounty Issue is encountering hardships that could not have been foreseen when work on it started; however, it is now in its final stages.

**Comment by AceTheCreator** [comment]
I added the changes you suggested @derberg, let me know what you think.

https://drive.google.com/file/d/1XS5NuMg0SjVLqDc5ylSgaMunTq7I9e3F/view?usp=sharing

**Comment by derberg** [comment]
😍 😍 😍 😍 😍 

job done like a pro!

you do not promote how to reach you - but it's your call

also, double check with Quetzalli we can use the picture at the end of the video

**Comment by AceTheCreator** [comment]
> 😍 😍 😍 😍 😍
> 
> job done like a pro!
> 
> you do not promote how to reach you - but it's your call
> 
> also, double check with Quetzalli we can use the picture at the end of the video

Thanks, man. 

I'll check with Quetzalli, as requested. 

Couldn't factor in how to promote how to reach me, cuz i didn't consider that during the initial planning of the video. 

I'm currently working on video #3 and it'll be included in it and future videos.

**Comment by aeworxet** [comment]
@derberg

Can this Bounty Issue be considered completed, or should the matter of whether Quetzalli would allow the use of her picture at the end of the video be clarified first, given that there is a minimal possibility that @AceTheCreator may still need to rework the last frames?

**Comment by AceTheCreator** [comment]
@aeworxet @derberg, So i was able to reach Quetzalli, and she was happy with it and in her words ` I feel so honored`

**Comment by derberg** [comment]
yeah, just a link below your name where you show `Ace Developer Advocate`

anyway, well done, job completed from my point of view

**Comment by aeworxet** [comment]
After a delay caused by technical hardships and a consent issue that could not be foreseen by the Bounty Program Participant
https://github.com/asyncapi/training/issues/41#issuecomment-2697490614
https://github.com/asyncapi/training/issues/41#issuecomment-2777961944
https://github.com/asyncapi/training/issues/41#issuecomment-2794710971

### Bounty Issue Is Completed 🎉

@AceTheCreator (`githubID`: [40604284](https://api.github.com/user/40604284)), please go to the [AsyncAPI page on Open Collective](https://opencollective.com/asyncapi) and submit an invoice for `USD 400.00` with the expense title `Bounty training#41`, tag `bounty`, and full URL of this Bounty Issue in the description.
